patches/server/0764-Added-ability-to-control-player-s-insomnia-and-phant.patch

```diff
@@ -0,0 +1,143 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jan Villim <jan@villim.sk>
+Date: Thu, 26 Aug 2021 23:39:33 +0200
+Subject: [PATCH] Added ability to control player's insomnia and phantoms
+ spawning in paper config
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index 2e0191e5cfe8e29fb0a6c4fc6a2a570d4b8ae449..41c96e7755685c82c9657e8a75355969e336f96f 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -289,6 +289,19 @@ public class PaperConfig {
+         suggestPlayersWhenNullTabCompletions = getBoolean("settings.suggest-player-names-when-null-tab-completions", suggestPlayersWhenNullTabCompletions);
+     }
+ 
++    public static int phantomsSpawnAttemptMinSeconds = 60;
++    public static int phantomsSpawnAttemptMaxSeconds = 119;
++    private static void insomniaControlSettings() {
++        phantomsSpawnAttemptMinSeconds = getInt("settings.insomnia-control.phantoms-spawn-attempt-min-seconds", phantomsSpawnAttemptMinSeconds);
++        phantomsSpawnAttemptMaxSeconds = getInt("settings.insomnia-control.phantoms-spawn-attempt-max-seconds", phantomsSpawnAttemptMaxSeconds);
```

Comment:
Have these settings be the same except at the end have `.min` and `.max`. I think that would look nicer. So like
`settings.insomnia-control.phantom-spawn-attempt-seconds.min`